sources/packages/api/src/services/application/application.service.ts

```diff
@@ -1343,4 +1343,25 @@ export class ApplicationService extends RecordDataModelService<Application> {
       )
       .getOne();
   }
+
+  /**
+   * Supporting user (e.g. parent/partner) formName
+   * of an application.
+   * @param applicationId application id.
+   * @returns application the application that was found, otherwise null.
+   */
+  async getSupportingUserFormName(applicationId: number): Promise<Application> {
```

Comment:
I believe that this method would be not necessary or it should be at least more generic and not related supporting user only.